src/pages/ProfilesExplorerView/components/SceneByVariableRepeaterGrid/SceneByVariableRepeaterGrid.tsx

```diff
@@ -68,29 +72,58 @@ const DEFAULT_SORT_ITEMS_FN: SceneByVariableRepeaterGridState['sortItemsFn'] = f
   return 0;
 };
 
-export class SceneByVariableRepeaterGrid extends SceneObjectBase<SceneByVariableRepeaterGridState> {
-  static DEFAULT_LAYOUT = LayoutType.GRID;
-
-  static buildGridItemKey(item: GridItemData) {
-    return `grid-item-${item.index}-${item.value}`;
-  }
-
-  static limitNumberOfSeries = () => (source: Observable<DataFrame[]>) =>
+export const DATA_TRANSFORMATIONS = {
+  addRefId: () => (source: Observable<DataFrame[]>) =>
+    source.pipe(map((data: DataFrame[]) => data.map((d, i) => merge(d, { refId: `${d.refId}-${i}` })))),
+  addStats: () => (source: Observable<DataFrame[]>) =>
     source.pipe(
       map((data: DataFrame[]) => {
         const totalSeriesCount = data.length;
 
-        return data.slice(0, LabelsDataSource.MAX_TIMESERIES_LABEL_VALUES).map((d, i) =>
-          merge(d, {
-            // adding unique "refId" and "totalSeriesCount" for overrides (see setupOverrides() below)
-            refId: `${d.refId}-${i}`,
+        return data.map((d) => {
+          const allValuesSum = d.fields
+            ?.find((field) => field.type === 'number')
+            ?.values.reduce((acc: number, value: number) => acc + value, 0);
+
+          return merge(d, {
             meta: {
-              stats: [{ displayName: 'totalSeriesCount', value: totalSeriesCount }],
+              stats: [
+                {
+                  displayName: 'totalSeriesCount',
+                  value: totalSeriesCount,
+                },
+                {
+                  displayName: 'allValuesSum',
+                  value: allValuesSum,
+                },
+              ],
             },
-          })
-        );
+          });
+        });
       })
-    );
+    ),
+  sortSeries: () => (source: Observable<DataFrame[]>) =>
+    source.pipe(
+      map((data: DataFrame[]) =>
```

Comment:
Sorting them for bar gauges but also for timeseries: if there are more than 10, sorting them here ensures that we always display the ones that consume more resources.